cmd/pinniped-proxy/src/cli.rs

```diff
@@ -0,0 +1,19 @@
+use structopt::StructOpt;
+
+#[derive(StructOpt, Debug)]
+/// A proxy server which converts k8s API server requests with bearer tokens to
+/// requests with short-lived X509 certs exchanged by pinniped.
+///
+/// pinniped-proxy proxies incoming requests with an `Authorization: Bearer
+/// token` header, exchanging the token via the pinniped aggregate API for x509
+/// short-lived client certificates, before forwarding the request onwards
+/// to the destination k8s API server.
```

Comment:
The triple slash comments are documentation comments. You'll see that the above is reformatted and used in the output of the binary with the `-h` help option.